docs/resources/client/textures/atlases.md

```diff
@@ -0,0 +1,98 @@
+# Texture Atlases
+
+Many textures in Minecraft are compiled within Atlases (also known as sprite sheets) for optimisation purposes. Most of them are automaticaly generated by Minecraft when registered throught the use of datapacks or [Registries][reg] (such as block textures).
+
+## Texture Atlas Sources
+
+Texture atlas sources are json files located inside the `atlases` asset folder.
+A texture atlas source describes the name of the targeted atlas and its contents. Additions to the atlases need to be created under the namespace of a mod or datapack.
+
+As of today 14 different atlas sources can be declared:
+
+|File name|Description|
+|--|--| 
+|`blocks`|Block textures|
+|`banner_patterns`|Banner patterns|
+|`beds`|Every bed variant (i.e. colors)|
+|`chest`|Chest textures|
+|`shield_patterns`|Shield textures and patterns|
+|`shulker_boxes`|Every shulker related textures (Shulker box variants, head and projectile)|
+|`signs`|Sign textures|
+|`particles`|Particle textures|
+|`paintings`|Every painting|
+|`mob_effects`|Potion effect icons|
+|`gui`|Every GUI wigets (buttons, scroll bars...)|
+|`map_decorations`|Icons displayed on a map (banners, locations...)|
+|`armor_trims`|Armor trim patterns for every material|
+|`decorated_pot`|Texture for the decorated pot and every sherd|
+
+### Creating an atlas source
+
+To add new textures to an atlas, a new file needs to be created in the `atlases` asset folder: `resources/assets/example_mod/atlases`.
+
+The file named the same as the altas it is appending to, will then describe what file will be added to the atlas:
+```json
+-- Inside gui.json
+{
+  "sources": [
+    -- Gather all sprites within the "source" folder
+    {
+      "type": "minecraft:directory",
+      "prefix": "",
+      "source": "gui/sprites/widgets"
+    },
+    -- Gather sprites within the "icons" spritesheet
+    {
+      "type": "minecraft:unstitch",
+      "resource": "example_mod:gui/sprites/icons",
+      "divisor_x": 0.0,
+      "divisor_y": 0.0,
+      "regions": [
+        {
+          "sprite": "example_mod:energy_low",
+          "x": 0.0,
+          "y": 0.0,
+          "width": 8.0,
+          "height": 8.0
+        }
+      ]
+    }
```

Comment:
Same thing here. What is this doing? All I understand reading this is that it attempts to merge the texture generated for the icon atlas with your custom one. x, y and the like are not explained.